python/src/aac/plugins/check/check_aac_impl.py

```diff
@@ -134,8 +132,15 @@ def check_schema_constraint(
 
             if field_definining_schema[0].get_root_key() == "primitive":
                 # if the field is a primitive, run the primitive constraints
+
                 if is_list:
                     # if the field is a list, check each item in the list
+                    if type(getattr(check_me, field.name)) != list:
+                        raise LanguageError(
+                            f"Value of '{field.name}' was expected to be list, but was '{type(getattr(check_me, field.name))}'. File: '{source_definition.source.uri}'",
+                            # source_definition.source.uri
```

Comment:
Remove this commented out code snippet. 
```suggestion
```

Reply:
Testing this out on my end, but this should be about it long as things work the way they were when we were shoulder surf working it Friday.